### DISCUSSION TITLE
Is it possible to add support for p281 boards like the tanix tx3 mini?

### DISCUSSION THREAD

**Discussion by Abdalnablse10** [johang/sd-card-images#78] [General]
It would be lovely to have pre-build images for p281 boards, also i don't know why and I'm not even sure but it isn't possible "for me at least" to boot without a boot partition with just a boot directory.

**Comment by johang** [comment]
> It would be lovely to have pre-build images for p281 boards

Looks like this is not supported by mainline U-Boot.

>  also i don't know why and I'm not even sure but it isn't possible "for me at least" to boot without a boot partition with just a boot directory.

Not sure what this means.

**Reply by Abdalnablse10** [reply]
Oh ok, sorry for the bothering.
